app/controllers/users/registrations_controller.rb

```diff
@@ -5,4 +5,9 @@ def build_resource(hash = {})
     hash[:uid] = User.create_unique_string
     super
   end
+
+  private
+    def after_update_path_for(resource)
+      user_path (resource)
```

Comment:
```suggestion
      user_path(resource)
```

半角スペースは無しで詰めておきましょう 👀